array-manipulation-2/src/drop-right.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+/* exported dropRight */
+function dropRight(array, count) {
+  const newArray = [];
+  for (let i = 0; i < array.length - count; i++) {
+    if (array[count]) {
```

Comment:
This check isn't necessary. In fact, thanks to some unexpected behavior with how JavaScript handles truthy/falsy, this actually will break. For example, if `array[count] === 0` or `array[count] === ''` then the element won't be pushed to the new array.